### DISCUSSION TITLE
请问G2考虑过增加使用大语言模型+自然语言来生成图形的功能吗？

### DISCUSSION THREAD

**Discussion by bigfacewo** [antvis/G2#5193] [Ideas]
现在Chatgpt挺火的，应用也挺多，比如[VizGPT](https://www.vizgpt.ai/)这个例子，就是使用自然语言来生成图形，只不过使用的图形库是vega-lite。

不知道G2这边有没有考虑这样的功能？

**Comment by pearmini** [comment]
目前还没有这个规划，但是这个能力长远来看确实需要，有兴趣用 G2 搞一个类似的项目吗？😝

**Reply by bigfacewo** [reply]
倒是有这个想法，不过如果涉及到自己训练模型，这就麻烦了一些。
